docs/sources/datasources/prometheus.md

```diff
@@ -27,6 +27,23 @@ To access Prometheus settings, hover your mouse over the **Configuration** (gear
 | `Disable metrics lookup`  | Checking this option will disable the metrics chooser and metric/label support in the query field's autocomplete. This helps if you have performance issues with bigger Prometheus instances.           |
 | `Custom Query Parameters` | Add custom parameters to the Prometheus query URL. For example `timeout`, `partial_response`, `dedup`, or `max_source_resolution`. Multiple parameters should be concatenated together with an '&amp;'. |
 
+### Exemplars
+
+> **Note:** This feature is available in Prometheus 2.25+.
+
+The Exemplars configuration allows you to add multiple endpoints to any of your Exemplar label.
```

Comment:
This looks a little bit out of place. Perhaps we can introduce exemplars after the introduction. Then add them into the various sections.